src/blocks/invention/fsh-api-animal.js

```diff
@@ -3,7 +3,7 @@ import Blockly from "blockly/core";
 const blockName = "inv_fsh_api_animal";
 
 const blockData = {
-  "message0": "get %1 image from fsh api",
+  "message0": "gets %1 of the possible images from the fsh api",
```

Comment:
Checking sementic from https://github.com/scratch-for-discord/Web-Application_Frontend/discussions/203

Reply:
![image](https://user-images.githubusercontent.com/69337718/224510069-99511cd3-5ee4-4413-a0f7-6674a5d8326e.png)
the block makes less sense now?
it grabs a random image of the specified animal

Reply:
i just noticed wait

Reply:
fixed

Reply:
https://github.com/scratch-for-discord/Web-Application_Frontend/pull/202#discussion_r1133144296 makes a bit more sense to me